README.md

```diff
@@ -3,13 +3,13 @@
 
 kampos lets you add filter effects and beautiful transitions to video and images (or any other media).
 Just like [SVG filter effects](https://developer.mozilla.org/en-US/docs/Web/SVG/Tutorial/Filter_effects),
-only using WebGL, and hence works everywhere!
+only using WebGL, which means it works everywhere!
 
 ## Demo
-Watch a [live demo](https://wix-incubator.github.io/kampos/demo/).
+Check out a [live demo](https://wix-incubator.github.io/kampos/demo/).
```

Comment:
Better, but then I think it should be: "Check out _the_", right?

Reply:
Yes, great catch!

Reply:
Addressed by 898519fcdf42be98c3c1c40a4eb8bf2e6f292a46, resolving.